Automattic/wp-calypso
# Add/warning in restore if backup is running

#### Proposed Changes

This change adds a change in the restores. If you want to do a restore and there is a backup running at that moment, you will see a warning message, and if you accept to do the restore, the backup will be stopped.

![Annotation on 2022-10-03 at 20-28-56](https://user-images.githubusercontent.com/2747834/193898487-3763760e-9d9e-4669-894f-9d4baaf92ac0.png)


#### Testing Instructions

While a backup is in execution, go to "Jetpack/Backup" and select a backup to do a restore. You will see a warning. Start the restore, and after some seconds, the restore will start.


#### Pre-merge Checklist






Related to #
